docs/content/preview/releases/yba-releases/v2.17.md

```diff
@@ -564,7 +564,7 @@ docker pull yugabytedb/yugabyte:2.17.0.0-b24
 * [PLAT-6042] Fix kubernetes CPU usage graph
 * [PLAT-6053] [PLAT-6054] Enable toggling force delete for xCluster config
 * [PLAT-6069] Fix: Incremental Backup location same as Full Backup Location
-* [PLAT-6074] Decrypt against all key versions for a key in azure KMS
+* [PLAT-6074] Decrypt against all key versions for a key in Azure Key Vault
```

Comment:
I see a bunch of duplicate entries in the file. It does not relate to your changes , but we have to get rid of the duplicates - Look at lines 143,144, 150,151 and such others. Maybe in another PR.